### ISSUE TITLE
Stack overflow with very simple Django templates

### ISSUE THREAD

**Issue by golinski** [jgm/pandoc#4431]
When formatting the following very simple file:

```
~~~ {.djangotemplate}
{% if var %}
string
{% endif %}
~~~
```

I get the following errors:

```
$ pandoc-amd64-7d193b2aa/pandoc test.md
pandoc: Stack space overflow: current size 33624 bytes.
pandoc: Use `+RTS -Ksize -RTS' to increase it.
$ pandoc-amd64-7d193b2aa/pandoc +RTS -K1000000000 -RTS test.md
pandoc: Stack space overflow: current size 33624 bytes.
pandoc: Use `+RTS -Ksize -RTS' to increase it.
```
The second command takes significantly longer to finish. This is from in the development version of pandoc from 9 hours ago.

**Comment by mb21** [comment]
Thanks, I assume the difference goes away with `--no-highlight`?

If so, this should be reported to https://github.com/jgm/skylighting

**Comment by jgm** [comment]
I opened an issue in jgm/skylighting.

**Comment by jgm** [comment]
OK, this is fixed if you compile against skylighting 0.7.0.2.

**Comment by Arezohayeman** [comment]
183ab26b4585238f49fd
